JulioC-19/LargeProjectMobile
# Julio/authentication

Implemented authentication. 

To test use: 
email: johnsmith@woah.com
password: ucf

1. Attempt to login with the credentials. 
2. After successfully login in, you should see the loading spinner while the app fetches user data. 
3. The app should navigate to the main screens. 
4. Close the app and open it again to get back to login screen. 

Note: 
I still need to implement an error message for invalid username and password and alert for successful sign up.
